pkg/configure/oneagent/pmc/ruxit/types.go

```diff
@@ -35,12 +58,53 @@ func (pmc ProcConf) ToMap() ProcMap {
 	return sections
 }
 
-// ToString creates the content of the configuration file, the sections and properties are printed in a sorted order, so it can be tested.
-func (pmc ProcConf) ToString() string {
-	rawMap := pmc.ToMap()
+// ProcMap presents the content in a more easy to work with format. (a map of maps).
+type ProcMap map[string]map[string]string
 
+var (
+	redundantEntries = map[string][]string{
+		"general": {"logDir", "dataStorageDir"},
+	}
+	additionalEntries = ProcMap{
+		"general": map[string]string{
+			"storage": "\"/var/lib/dynatrace/oneagent\"", // TODO: Make configurable?
+		},
+	}
+)
+
+func (pm ProcMap) SetupReadonly(installPath string) ProcMap {
+	for key, values := range redundantEntries {
+		_, ok := pm[key]
+		if !ok {
+			continue
+		}
+
+		for _, value := range values {
+			delete(pm[key], value)
+		}
+	}
+
+	for section, entries := range pm {
+		for entry, value := range entries {
+			if strings.Contains(value, "../") {
```

Comment:
from which source we can get `../` style of dir? can we use [Base](https://pkg.go.dev/path/filepath#Base) or so?

Reply:
not really:
- not all values are paths (only the paths will have `"../"`) (so we can't do `!filepath.IsAbs(value)`)
- it is not a "clean" filepath, it has `"` around it so `\"../../path/file.txt\"`

Reply:
now i got it, so ignore it